core/src/trezor/ui/layouts/tt/__init__.py

```diff
@@ -1025,6 +1030,60 @@ async def confirm_ethereum_tx(
             continue
 
 
+async def confirm_ethereum_staking_tx(
+    title: str,
+    intro_question: str,
+    verb: str,
+    total_amount: str,
+    maximum_fee: str,
+    address: str,
+    address_title: str,
+    info_items: Iterable[tuple[str, str]] | None = None,
+    chunkify: bool = False,
+    br_type: str = "confirm_ethereum_staking_tx",
+    br_code: ButtonRequestType = ButtonRequestType.SignTx,
+) -> None:
+
+    # intro
+    # NOTE: this layout very similar to `confirm_value` with some adjustments
+    msg_layout = RustLayout(
+        trezorui2.confirm_value(
+            title=title,
+            value=intro_question,
+            description=None,
+            subtitle=None,
+            verb=verb,
+            info_button=True,
+            text_mono=False,
+        )
+    )
+    info_layout = RustLayout(
+        trezorui2.show_info_with_cancel(
```

Comment:
Are there some tests of clicking the info button and seeing the details? Quickly looked at some test in CI and did not see any - https://satoshilabs.gitlab.io/-/trezor/trezor-firmware/-/jobs/6216595229/artifacts/test_ui_report/index.html

Reply:
Yes, good catch in these two comments. 
I will use chunkify for Staking/Claiming address screens on TS3 and add Click tests which verify it.

Reply:
Clicking the info button added
https://github.com/trezor/trezor-firmware/pull/3534/commits/6f395b9859bf10da578a889310ca4548c03a5d95